### ISSUE TITLE
Incorrect release date

### ISSUE THREAD

**Issue by Lifebario** [Lifebar/support#8]
Submitter: mamoru

The release date for Sam & Max Season 2: Beyond Time and Space is incorrect on the site:
http://lifebar.io/#game/25811/Sam+%26+Max+Beyond+Time+and+Space

The episodes were released between November 07 to April 08.
(ref: https://en.wikipedia.org/wiki/Sam_%26_Max_Beyond_Time_and_Space)

**Comment by rewphus** [comment]
Good catch.  I am not quite sure why it is currently showing that way in our system, but we will take a look.  If you find any more info that appears inaccurate, please let us know.
